bin/ci/deploy-dockerhub.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
do we want to do `#!/bin/bash -ex` here?

Reply:
We already `set -e`, I've added `set -x` too.